modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/generalized_plane_strain.md

```diff
@@ -0,0 +1,125 @@
+# Generalized Plane Strain
+
+## Description
+
+The generalized plane strain problem has found use in many important applications. Bearing the same name as generalized plane strain problem, different presentations have been found in the literature. The simplest one is an extension of the plane strain problem by allowing a non-vanishing constant direction strain in the out-of-plane direction [citep:Adams1967gps]. This has been further generalized by allowing two rotations degrees of freedom [citep:Abaqus2014gps]. An even more generalized form includes the anticlastic problem associated with the out-of-plane shear [citep:Adams1984gps, Li1999gps]. In the [tensor mechanics module](tensor_mechanics/index.md), the simplest form was implemented, i.e. an extra degree of freedom representing the out-of-plane direct strain is included in addition to the conventional plane strain problem [citep:Li2005gps].
```

Comment:
- Delete the first clause in the  second sentence in this paragraph. (sentence should read "Many different presentations of the generalized plane strain problem are given in the literature.")
- Fourth sentence begins with a this that doesn't refer to a clear noun (eschew pronoun obfuscation!)
- Make the last sentence a stand alone paragraph